buildSrc/build.gradle.kts

```diff
@@ -1,79 +1,71 @@
 plugins {
-  `kotlin-dsl`
-
-  // When updating, update below in dependencies too
-  id("com.diffplug.spotless") version "6.25.0"
+    `kotlin-dsl`
+    // When updating, update below in dependencies too
+    id("com.diffplug.spotless") version "6.25.0"
 }
 
 if (!hasLauncherForJavaVersion(17)) {
-  throw GradleException(
-    "JDK 17 is required to build and gradle was unable to detect it on the system.  " +
-        "Please install it and see https://docs.gradle.org/current/userguide/toolchains.html#sec:auto_detection " +
-        "for details on how gradle detects java toolchains."
-  )
+    throw GradleException(
+        "JDK 17 is required to build and gradle was unable to detect it on the system. " +
+            "Please install it and see https://docs.gradle.org/current/userguide/toolchains.html#sec:auto_detection " +
+            "for details on how gradle detects java toolchains."
+    )
 }
 
 fun hasLauncherForJavaVersion(version: Int): Boolean {
-  return try {
-    javaToolchains.launcherFor { languageVersion = JavaLanguageVersion.of(version) }.get()
-    true
-  } catch (e: Exception) {
-    false
-  }
+    return try {
+        javaToolchains.launcherFor { languageVersion.set(JavaLanguageVersion.of(version)) }.get()
+        true
+    } catch (e: Exception) {
+        false
+    }
 }
 
 spotless {
-  kotlinGradle {
-    ktlint().editorConfigOverride(mapOf(
-      "indent_size" to "2",
-      "continuation_indent_size" to "2",
-      "max_line_length" to "160",
-      "insert_final_newline" to "true",
-      "ktlint_standard_no-wildcard-imports" to "disabled",
-      // ktlint does not break up long lines, it just fails on them
-      "ktlint_standard_max-line-length" to "disabled",
-      // ktlint makes it *very* hard to locate where this actually happened
-      "ktlint_standard_trailing-comma-on-call-site" to "disabled",
-      // depends on ktlint_standard_wrapping
-      "ktlint_standard_trailing-comma-on-declaration-site" to "disabled",
-      // also very hard to find out where this happens
-      "ktlint_standard_wrapping" to "disabled"
-    ))
-    target("**/*.gradle.kts")
-  }
+    kotlinGradle {
+        ktlint().editorConfigOverride(mapOf(
+            "indent_size" to "2",
+            "continuation_indent_size" to "2",
+            "max_line_length" to "160",
+            "insert_final_newline" to "true",
+            "ktlint_standard_no-wildcard-imports" to "disabled",
+            "ktlint_standard_max-line-length" to "disabled",
+            "ktlint_standard_trailing-comma-on-call-site" to "disabled",
+            "ktlint_standard_trailing-comma-on-declaration-site" to "disabled",
+            "ktlint_standard_wrapping" to "disabled"
+        ))
+        target("**/*.gradle.kts")
+    }
```

Comment:
the comments that were here before were important. Please restore this file to what it was before.